src/redux/app/actions.js

```diff
@@ -9,7 +9,6 @@ export function getView(width) {
 }
 
 const actions = {
-  COLLPSE_CHANGE: 'COLLPSE_CHANGE',
   COLLPSE_OPEN_DRAWER: 'COLLPSE_OPEN_DRAWER',
```

Comment:
Can we get rid of this one too? Seems like the drawer related logic is the deleted from the sidebar.

Reply:
okay, will do a grep and remove it.